### ISSUE TITLE
react-avatar@2.5.1: The engine "node" is incompatible with this module.

### ISSUE THREAD

**Issue by RathanakSreang** [Sitebase/react-avatar#78]
I try to use yarn to install react-avatar.
`yarn install`
Then it show error message:
```
yarn install v0.27.5
warning package.json: No license field
warning No license field
[1/4] Resolving packages...
[2/4] Fetching packages...
error react-avatar@2.5.1: The engine "node" is incompatible with this module. Expected version "8.x.x".
error Found incompatible module
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
```
my node version is: `v9.2.0`
Is there any solution without change my current node version?

**Comment by loris** [comment]
Maintainers nees to update their `package.json`, meanwhile you can use `yarn install --ignore-engines`

**Comment by homejames203** [comment]
I'm getting the same error & would really like to continue using this library. Can the project be updated to not require a specific node version when using the library?

**Comment by allanhortle** [comment]
Got a blunt fix here: #80

**Comment by twang2218** [comment]
I'm also affected by this bug, my node version is `v9.4.0`. It's very annoying to add `--ignore-engines` for every `yarn` command you run. Please fix this bug, and merge #80.

**Comment by JorgenEvens** [comment]
Resolved by https://github.com/Sitebase/react-avatar/pull/80
